Src/libCZI/decoder_wic.h

```diff
@@ -6,7 +6,7 @@
 
 #include <libCZI_Config.h>
 
-#if LIBCZI_WINDOWSAPI_AVAILABLE
+#if LIBCZI_WINDOWSAPI_AVAILABLE && LIBCZI_HAVE_WINCODECS_API
```

Comment:
I don't see how this define would be propagated into the compilation. I'd don't see that the [libCZI_Config.h.in](https://github.com/ZEISS/libczi/blob/main/Src/libCZI/libCZI_Config.h.in) mechanism is used - which would be the preferred way I guess).